package sgpiv.dtos.request;

import jakarta.validation.constraints.NotBlank;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TareaSoliDTORequest {

    @NotBlank
    private String titulo;

    @NotBlank
    private String descripcion;

}
